examples/hil.cfg.dev-no-hardware

```diff
@@ -26,20 +26,9 @@ dry_run=True
 [extensions]
 hil.ext.switches.mock =
 hil.ext.obm.mock =
-
 hil.ext.network_allocators.null =
-
-# Depending on what you're trying to do, you may want to use the vlan_pool
-# network allocator instead of the null allocator. To do this, comment out the
-# null allocator extension above, and uncomment the following:
-#
-# hil.ext.network_allocators.vlan_pool =
-#
-# [hil.ext.network_allocators.vlan_pool]
-# vlans = 100-200, 300-500
-
 hil.ext.auth.null =
-# Again, depending on what you're doing, you may want to change this to a
+# Depending on what you're doing, you may want to change this to a
 # diffefrent authentication backend:
```

Comment:
`diffefrent ...`
Let's fix this spelling error while we're at it.

Reply:
@SahilTikale, do this and I'll hit merge; this looks fine to me.